### DISCUSSION TITLE
IoT Inspector Client for macOS Sonoma 14.2

### DISCUSSION THREAD

**Discussion by CTiNb39** [noise-lab/iot-inspector-client#220] [General]
I used the Princeton IoT Inspector 2019 on a MacBook Pro. Installation was 10 minutes.  Worked the first time.  

Information for macOS IoT Inspector 2 is not sufficient (at least for me). Can someone point me in the right direction?

**Comment by NikSWE** [comment]
could you share the specific errors you're getting while building from source. I tested it on MacOS Ventura and didn't face any build problems.

**Comment by crazyideas21** [comment]
> Information for macOS IoT Inspector 2 is not sufficient (at least for me). Can someone point me in the right direction?

https://github.com/nyu-mlab/iot-inspector-client/wiki/Download-&-Install#macos @CTiNb39
